src/components/Rockets/elementRockets.js

```diff
@@ -0,0 +1,52 @@
+import { useDispatch } from 'react-redux';
+import Button from 'react-bootstrap/Button';
+import { PropTypes } from 'prop-types';
+import styles from '../../pages/Rockets.module.css';
+import { addRocketReservation, removeRocketReservation } from '../../redux/rockets/rockets';
+
+const elementRockets = (props) => {
+  const dispatch = useDispatch();
+  const {
+    id, name, image, desc, reserved,
+  } = props;
+  // console.log(desc);
```

Comment:
- [ ] Kindly remove the commented code so your code is clean.

Reply:
Asked change implemented

Reply:
All good now 👍🏽